### DISCUSSION TITLE
Why do each of you maintain a Solid.js version of shadcn/ui?

### DISCUSSION THREAD

**Discussion by LiMao00** [hngngn/shadcn-solid#69] [General]
Why do each of you maintain a Solid.js version of shadcn/ui?
https://www.solid-ui.com/
https://github.com/sek-consulting/solid-ui

and
https://shadcn-solid.com/
https://github.com/hngngn/shadcn-solid

**Comment by hngngn** [comment]
Honestly, I don't know how to answer this. When I started this project, I didn't know about the Sek project until 2-3 months later. But since I've already put a lot into this, I'll just keep going.

**Reply by LiMao00** [reply]
Anyway, thank you for your efforts.
